frontend/src/views/housing/HousingView.vue

```diff
@@ -1,105 +1,251 @@
-  //   route: RouteName.HOUSING_PROJECTS_LIST
-  // }
-]);
+import type { Ref } from 'vue';
+import type { Enquiry, Submission } from '@/types';
+import EnquiryListProponent from '@/components/housing/enquiry/EnquiryListProponent.vue';
+
+// Constants
+const PAGE_ROWS = 5;
+
+// State
+const drafts: Ref<Array<any>> = ref([]);
+const enquiries: Ref<Array<Enquiry>> = ref([]);
+const projects: Ref<Array<Submission>> = ref([]);
+const first: Ref<number> = ref(0);
+const displayedProjects = computed(() => projects.value.slice(first.value, first.value + PAGE_ROWS));
+const loading: Ref<boolean> = ref(true);
+
+// Actions
+const { t } = useI18n();
+const router = useRouter();
+
+function onProjectClick(project: Submission) {
+  router.push({
+    name: RouteName.HOUSING_SUBMISSION_INTAKE,
+    query: { activityId: project.activityId, submissionId: project.submissionId }
+  });
+}
+
+function onSubmissionDraftDelete(draftId: string) {
+  drafts.value = drafts.value.filter((x) => x.draftId !== draftId);
+}
+
+onMounted(async () => {
+  [enquiries.value, projects.value, drafts.value] = (
+    await Promise.all([
+      enquiryService.getEnquiries(),
+      submissionService.getSubmissions(),
+      submissionService.getDrafts()
+    ])
+  ).map((r) => r.data);
+
+  drafts.value = drafts.value.map((x, index) => ({ ...x, index: index + 1 }));
+
+  loading.value = false;
+});
 </script>
 
 <template>
   <div class="flex flex-column align-items-center justify-content-start h-full">
-    <div class="flex flex-row align-items-center w-full housing">
+    <div class="flex flex-row align-items-center w-full justify-content-between shadow-2 px-3 py-1">
+      <h1>Housing</h1>
       <img
         class="mr-3"
-        src="@/assets/images/house.png"
-        width="67"
-        height="64"
+        src="@/assets/images/housing_2.png"
+        width="120"
         alt="Housing image"
       />
-      <h1>Housing</h1>
     </div>
-    <div class="flex flex-column align-items-center justify-content-start mt-8">
-      <div
-        v-for="(item, index) in items"
-        :key="index"
-        class="flex justify-content-center mb-5"
-      >
+
+    <div class="flex flex-row align-items-center w-full mt-3">
+      <div class="font-bold mr-2">{{ t('housing.onThisPage') }}</div>
+      <!-- eslint-disable vue/multiline-html-element-content-newline -->
+      <!-- prettier-ignore -->
+      <div>
+        <router-link
+          :to="{
+            name: RouteName.HOUSING,
+            hash: '#projects'
+          }"
+          class="no-underline"
+        >
+        {{ t('housing.myProjects') }}</router-link>
+        |
         <router-link
-          v-slot="{ href, navigate }"
-          :to="{ name: item.route }"
-          custom
+          :to="{
+            name: RouteName.HOUSING,
+            hash: '#drafts'
+          }"
+          class="no-underline"
+        >
+        {{ t('housing.drafts') }}</router-link>
+        |
+        <router-link
```

Comment:
Style: If we're attempting to create horizontal bars, this might be doable via CSS/Class `border-left` property or equivalent in lieu of using the pipe character. That might be able to make it so we don't need to use eslint and prettier ignores for this block of code.